packages/block-library/src/pullquote/style.scss

```diff
@@ -69,7 +73,3 @@
 		}
 	}
 }
-
-.wp-block-pullquote cite {
```

Comment:
Any explanation why this CSS is being removed?

Reply:
Hey @akasunil,

The reason for removing this CSS is that the same styles are already being applied through the following selector on line 11:
```css
p,
blockquote,
cite {
     color: inherit;
}
```

here’s the relevant section in the [file](https://github.com/SainathPoojary/gutenberg/blob/fix/pullquote-spacing/packages/block-library/src/pullquote/style.scss#L7-L13):

Reply:
Make sense!